### DISCUSSION TITLE
Mirror Validation Rules to input in Blade

### DISCUSSION THREAD

**Discussion by ProgrammerZ** [laravel/framework#52216] [Ideas]
It would be awesome not to copy common validation attributes to the `<input>` like: `required`, `minlength`, `maxlength` and `pattern`.

Let's say we have a rule  `'title' => 'required|unique:posts|max:255'`
and by providing something like `<input type="text" @rules('title') />`
it will output `<input type="text" required maxlength="255">`

**Comment by henzeb** [comment]
That's why they built precognition. 😅☺️ https://laravel.com/docs/11.x/precognition

**Reply by ProgrammerZ** [reply]
@henzeb Thanks. In my case I use Livewire and using Precognition is just an overkill for this )). This functionality may just expand pure blade functionality like `@error`, `@required` and etc.
